packages/ui/src/components/AppContent/index.tsx

```diff
@@ -29,9 +29,18 @@ const AppContent = ({ children }: Props) => {
     document.body.classList.add(styles[theme] ?? '');
   }, [theme]);
 
+  useEffect(() => {
+    if (isMobile) {
+      document.body.classList.add(styles.mobile ?? '');
+
+      return;
+    }
+    document.body.classList.add(styles.desktop ?? '');
```

Comment:
```suggestion
    document.body.classList.add(
      conditionalString(isMobile && styles.mobile),
      conditionalString(!isMobile && styles.desktop)
    );
```

Reply:
<img width="864" alt="image" src="https://user-images.githubusercontent.com/36393111/167751442-6f0b1464-ec8c-4129-ac03-a629a70881ac.png">

Looks like empty string is not allowed

Reply:
updated using the conditional string however keep the is condition statement